manager/orchestrator/replicated/update_test.go

```diff
@@ -204,6 +211,12 @@ func testUpdaterRollback(t *testing.T, rollbackFailureAction api.UpdateConfig_Fa
 	assert.Equal(t, observedTask.Status.State, api.TaskStateNew)
 	assert.Equal(t, observedTask.Spec.GetContainer().Image, "image1")
 
+	if !setMonitor {
```

Comment:
The test checks that the rollback is triggered (see the check for `UpdateStatus_ROLLBACK_STARTED` above). The rest of the test is about what happens after the rollback starts, and I wouldn't have included this in a test designed to catch this regression, but wanted to reuse the code of this existing test, so I made it return here for this new testcase. Does that sound okay?